### DISCUSSION TITLE
Startup imports unkown

### DISCUSSION THREAD

**Discussion by bdch1234** [microsoft/pylance-release#2918] [Enhancement]
Not sure if this should be filed here or in the Pylance-repo, but here goes.

Using the default startup functionality makes Pylance complain about unkowns. As an example, if I make a file startup.ipy, with the only contents "import numpy as np" in it, and put it in the "~/.ipython/profile_defaults/startup" directory, then I can in every new notebook directly use the np-import. Pylance however, will still give the errors "np is not defined".

**Comment by rchiodo** [comment]
Thanks for the issue. This would likely have to be handled by pylance (if at all). Transferring there.

**Comment by heejaechang** [comment]
we should put this file as the first chained file in the notebook so symbols defined in the file is implicitly imported and available to cells.
